server/posts.go

```diff
@@ -67,6 +68,36 @@ func (app App) GetPost(w http.ResponseWriter, req *http.Request) {
 	}
 }
 
+// GetPostImage redirects the client to an image URL
+// @summary Returns a redirect to an image URL
+// @router /posts/{id}/images/{filePath} [get]
+// @tags posts
+// @id getPostImage
+// @Param id path integer true "Post ID"
+// @Param imageFile path string true "Image file path"
+// @success 307 {header} string
+// @failure 404 {object} api.Error "Not found"
+// @failure 500 {object} api.Error "Server error"
+// @Security OAuth2Implicit[cms,openid,profile,email]
+// @Security OAuth2AuthCode[cms,openid,profile,email]
+func (app App) GetPostImage(w http.ResponseWriter, req *http.Request) {
```

Comment:
I'll modify this and the corresponding API function to accept height and/or width query parameters so clients can request thumbnails. We'll check if the thumbnail exists already and if not generate it before returning. (I've found that thumbnail generation is pretty fast.)